homeassistant/components/surepetcare/services.yaml

```diff
@@ -20,3 +20,23 @@ set_lock_state:
             - 'locked_in'
             - 'locked_out'
             - 'unlocked'
+
+set_pet_location:
```

Comment:
This adds a new service, however, I don't see a matching PR that documents this new service. Could you point me to the documentation PR @Danielhiversen?

Reply:
Here https://github.com/home-assistant/home-assistant.io/pull/19416 :flushed:

Reply:
Awesome! Thanks @Danielhiversen 👍